lib/utils/eventually-update.js

```diff
@@ -38,7 +38,7 @@ const npmUpdate = async (serverless, { newVersion, tarballUrl, abortHandler }) =
   try {
     await fse.remove(tempInstallationDir);
     await fse.ensureDir(tempInstallationDir);
-    const downloadStream = got.stream(tarballUrl);
+    const downloadStream = fetch(tarballUrl);
```

Comment:
@debadutta98 Are you sure about this change? Here its `got.stream` that was used to retrieve stream.

Unfortunately this part is not covered well by tests, so we need to be additionally carefully, best if somehow manually this change is tested

Reply:
``` javascript 
const videourl = 'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4'
const fs = require('fs');
const stream = require('stream');
const fetch = require('node-fetch');
const fsPromises = require('fs/promises');
const storeVideo = async (url) => {
  try {
  const response = await fetch(url);
  const fileStream = fs.createWriteStream('BigBuckBunny.mp4');
response.body.on('finish',()=>{
  console.log('video is successfully downloaded');
});
response.body.pipe(fileStream);
} catch(err) {
  console.error(err);
}
}
fs.stat('BigBuckBunny.mp4',async (err)=>{
  if(!err) {
    await fsPromises.unlink('BigBuckBunny.mp4')
  } else {
    console.log('file not found we are good to go now');
  }
  storeVideo(videourl);
});
```
@medikoo you can refer this code and [here](https://www.npmjs.com/package/node-fetch#streams) is the original example.

Reply:
@debadutta98 here it's `got.stream` not `got` that's originally used, and per documentation `fetch()` returns _promise_ not _stream_. So it doesn't look as valid counterpart